Medical Dispatch POI

dustylens opened the pull request:
<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR
<!-- What did you change? -->
<!-- If this is a code change, summarize at high level how your new code works. This makes it easier to review. -->

This PR adds a new POI to serve as a shipyard for medical craft and a drop off point for medical bounties. 

It also serves as an initial base for the Medical Operations Manager (MOM). A new whitelisted role that will serve as a testbed for rolling out medical pay systems in addition to serving as a public face for organizing medical play and incorporating new players into the field.

## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->

- Maintainer request for a new place to store medical ships and bounty drop off points.
- Maintainer request for a place to stash a new medical director to facilitate pay.

As per request this POI offers virtually no specific services and is not, in fact, a medical center. It is a place where players may spawn in, purchase a ship and vital round start equipment and then go on their merry way. It is a gathering place for medics and a good space to find and collaborate with peers. But it is not a hospital in its default form.

The whitelisted role is currently intended to serve as a guide and to facilitate cooperation by example and via provision of basic pay to selected cooperating medics. We're going to feel this one out as time goes on and adjust based on feedback and experience.

There will be one shift Mom, but the medic roles themselves remain pilot/mercenary picks.

Currently the station has spawn in points for pilots, mercenaries and the MOM (love you mom). 

There is no contractor spawn in point. This seemed appropriate as it might be more than a bit of a noob trap. Players who want to sign on with medical crews may do so through traditional means.

Mom access is currently based on the SR. This could probably be dropped to the STC based on ongoing discussion.
Mom loadouts are currently based on mercenary loadouts due to lack of specific role drip. That isn't something that I view as a blocker at present and not worth holding up the PR. CMO gear is provided in the spawn in room.

This PR is, I feel, currently at an acceptable MVP state and is worth examining for tweaks and changes to ensure that naming conventions, role permissions and other requirements are where they should be. This is a first run for me so I may have missed a few things.

The biggest to:do on my own end is largely aesthetics based. Which is going to go about as quickly as my own spiriting skills permit.

There are a lot of role and loadout changes here that are a first for me. So I anticipate that there will be some initial bugs to iron out. 

## How to test
<!-- Describe the way it can be tested -->

Fire it up. Try spawning in. Check out the station. Check out the roles. 

## Media
<!-- Attach media if the PR makes ingame changes (clothing, items, features, etc). 
Small fixes/refactors are exempt. Media may be used in SS14 progress reports with credit. -->
![ss+(2024-11-23+at+04 00 11)](https://github.com/user-attachments/assets/1d6cc174-dc84-44d9-bbb6-7db2e42f4a76)
![ss+(2024-11-23+at+04 01 59)](https://github.com/user-attachments/assets/f03db1f4-45d8-4715-876b-41c1fde9c716)
![ss+(2024-11-23+at+04 01 48)](https://github.com/user-attachments/assets/502b935c-a194-44fe-805d-1bcc4c5606ff)

## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
<!-- List any breaking changes, including namespaces, public class/method/field changes, prototype renames; and provide instructions for fixing them.
This will be posted in #codebase-changes. -->

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->
<!--
:cl:
- add: Added fun!
- remove: Removed fun!
- tweak: Changed fun!
- fix: Fixed fun!
-->
🆑 
- add: Added Medical Dispatch, a POI with spawn points.
- add: Added a new whitelisted command role, the Director of Care.
- tweak: Medical ships moved from Frontier to new medical dispatch POI.

dustylens committed:
Medical Dispatch POI

dustylens committed:
Remove mail metrics from MOM pda

dustylens committed:
Poster update. Always gotta have a poster.

dustylens committed:
Map orientation shift, poster update.

dustylens committed:
Replaces accidentally deleted default ID card.

dustylens committed:
Attempting to correct branch conflicts.

dustylens committed:
Attempt 2 to correct conflicts.

dustylens committed:
Attempt 3 - Remove posters.yml

It seems this file was removed from this location and poster spawners were moved to \Signs\contraband.yml

dustylens committed:
Merge branch 'master' into NF-Dispatch-2

dustylens committed:
Correcting errors from upstream merge MK I

Adjust lights, contraband poster signs.

dustylens committed:
Fixes to play time tracker and poster upstream merge code changes.

dustylens committed:
Hopefully final poster squash.

dustylens committed:
*drumroll*

dustylens committed:
comma

dustylens committed:
Fix the arbitrage spam.

dustylens committed:
More arbitrage issues. Let's squash them all.

dustylens committed:
Hopefully final upstream poster issue squash. What a thrill.

dustylens committed:
Removes hitchhiking contraband.yml

dustylens committed:
Merge branch 'master' into NF-Dispatch-2

dustylens committed:
Merge remote-tracking branch 'upstream/master' into NF-Dispatch-2

dustylens committed:
Conflict removal.

dustylens committed:
Ongoing conflict removals. Love it.

dustylens committed:
New wall sign and mom guidebook update. Bank prep phase.

dustylens committed:
Merge branch 'master' into NF-Dispatch-2

whatston3 committed:
Med bank acct, ledger entries, med bounty "tax"

dustylens committed:
Polished guidebook update.

dustylens committed:
Merge branch 'NF-Dispatch-2' of https://github.com/dustylens/frontier-station-14 into NF-Dispatch-2

whatston3 commented:
For what it's worth, I've written up a medical bank account, started "taxing" medical bounties (50% for now), added a modest stipend (18k/hr).  Replaced the mapped station admin console with a medical version - should be functionally set up for the new role.  All changes up on [whatston3:2024-11-28-NF-Dispatch-2-bank-n-bounty](https://github.com/new-frontiers-14/frontier-station-14/compare/master...whatston3:frontier-station-14:2024-11-28-NF-Dispatch-2-bank-n-bounty?expand=1)

dustylens committed:
Adds medical admin terminal and updates memorial graphic

dustylens committed:
Merge branch 'master' into NF-Dispatch-2

dustylens committed:
Paperwork for Whatstone to say "This is fine but" after reading

dustylens committed:
Straight up added a box of red/green light bulbs I don't care fight me.

dustylens committed:
Light boxes into the spawner pool. Finish your work dusty.

dustylens committed:
Saloon cleanup

dvir001 commented:
![image](https://github.com/user-attachments/assets/e9a13737-fd7d-4bfc-81d7-50a1c5536a73)

dustylens committed:
First pass fixes

Update job icon
Update job loadouts to be more CMO appropriate
Minor POI updates
Minor job name update

dustylens committed:
Merge branch 'master' into NF-Dispatch-2

dustylens committed:
Remove CMO kit from loadouts.

Slight POI changes.

Remove CMO gear from most places. Remove CMO stamp from CMO locker.
Add medical belt to CMO locker.

dustylens committed:
Merge branch 'master' into NF-Dispatch-2

BlueHNT commented:
Station with Ops manager, might add some points system for recovered bodies from those emergencies rather than the money. So, the manager can buy advanced topicals so you can focus on the actual healing, rather than spending 2 hours in space making a few stitches and whatnot.

whatston3 committed:
Merge branch 'master' of https://github.com/new-frontiers-14/frontier-station-14 into NF-Dispatch-2

whatston3 committed:
remove "whitelistRequired": "whitelisted" in job

whatston3 commented:
Minor point - in this hallway here, the radar consoles currently get a rotated view of the station, showing south as "up".  Might recommend swapping the position of the poster/map (doesn't have the same issue) and the radar consoles.  Nice use of wallmount radar consoles, by the way.

![image](https://github.com/user-attachments/assets/e8af65f7-62af-4507-a193-cc94e0fd4a87)

Might give the external airlocks dock numbers, but that can happen later.

I think the command blue makes sense for the job icon - the existing CMO icon seems ideal to me: denotes a position of command and associates with medical personnel via the cross, and I'd argue the ID card should be the same silver that the STC has (might give the SR/Sheriff gold IDs FWIW), presumably with a medical-coloured light blue insert along the bottom.

dvir001 commented:
Really need someone to review the map, the code side is ready.

dvir001 committed:
Remove whitespace

dvir001 reviewed (approved):
On my way to buy robux
![image](https://github.com/user-attachments/assets/ee7ce2c2-752a-4e85-b53e-9b338c4e0047)

Code side all good, require map pass.

dvir001 committed:
Merge branch 'master' into NF-Dispatch-2

dvir001 committed:
Update general.yml

dvir001 committed:
Merge branch 'master' into NF-Dispatch-2

dvir001 committed:
Update bar_sign.yml

dvir001 committed:
Update bar_sign.yml

dvir001 committed:
Update medical.yml

dustylens committed:
Don't just randomly delete stuff friend.

MagnusCrowe commented:
**There's too many lights. Pretty much across the entire station.  Here are a series of recommendations.**
Morgues: I used a single blue light to improve the "morgue ambience" and I replaced two of the morgue slabs with a scanner computer and an operating table...two props that are at home in the morgue, both offering visual contrast.
![image](https://github.com/user-attachments/assets/eadaae54-5a94-42cb-822b-46ecfe9b57bd)
Docks: Letting more of that green and red light in from outside the hallways and lowering the overall brightness throughout.
![image](https://github.com/user-attachments/assets/fa2a3889-ac7b-4d24-9220-3a199bf3dbe9)
Gardens:  I recommend using glowing plant life to replace some of the harsh bright tubes with some softer colored lights.  I put the tall tree on the side without the table to prevent rotations from hiding the table.
![image](https://github.com/user-attachments/assets/0483facc-c0f2-4b38-a947-99eb7c250885)
CMO Office, bridge, and south vending hall: These were mostly ok, I think I moved a light in the hallway around the corner and deleted the light over the computers in the bridge, allowing the light they give off to be seen better.
Construction area: Used to have 3 lights, replaced with a single sodium light.  Players can replace it or build new light fixtures themselves.  The sodium light gives off the "unfinished construction" vibe better than a big blast of white light.
![image](https://github.com/user-attachments/assets/a51f3965-3626-4a53-85c5-146983274bae)
Bar/Lounge: I replaced all the lights in this room with 2 orange lights.  It had a lot of cozy furniture and flooring, but very clinical and sterile lighting.  The orange lights fit the cozy furniture vibe and since there's only 2 of them, it's not overpowering (and it fits the fireplace light better).
![image](https://github.com/user-attachments/assets/bbbf6087-5948-478f-bd3f-ca4a926b41cf)
Here's what some areas look like from a player perspective.
![image](https://github.com/user-attachments/assets/7d865260-da89-4302-b77a-d20b9abc1e4e)
![image](https://github.com/user-attachments/assets/8cf2acf1-d4e5-4505-9f77-5aabc7c9b7fa)
![image](https://github.com/user-attachments/assets/788c8fc8-b931-43c8-bbf4-6b3c7cfd97ca)
![image](https://github.com/user-attachments/assets/cd904816-c3fa-4d24-8190-2c0c2500a8a2)
![image](https://github.com/user-attachments/assets/2d1fc660-882b-4ae8-a8d9-68c6ce02bea3)
![image](https://github.com/user-attachments/assets/6199b105-91b8-483a-bda2-ed4d1e18a86e)

dustylens committed:
Merge branch 'master' into NF-Dispatch-2

MagnusCrowe commented:
![image](https://github.com/user-attachments/assets/3001dc11-4349-4a41-8613-3e751f847461)
What about these areas are "Cargo"? 
![image](https://github.com/user-attachments/assets/46614e14-e6a1-4385-8e9b-1642b5d8fe02)

![image](https://github.com/user-attachments/assets/50f7a0f4-a263-44a3-8b1e-da5e46e5cebc)
Good job having the shuttle arrive at the station.

![image](https://github.com/user-attachments/assets/66160635-8276-4779-a942-c8e30297ad7c)
This room confuses me.  It's got treatment beds and cryo beds and a table and charger.  Is it a treatment room?  It should be bigger, have a medical locker, and maybe more table space. Is it a cryo bed room?  Why does it have all this extra treatment stuff in it.  It seems cramped and unfocused.

![image](https://github.com/user-attachments/assets/6c7da72b-dd22-416c-89f1-32d18ba4aa3c)
Janitorial/disposals looks good, only question is, why does it have a gas condenser? 

![image](https://github.com/user-attachments/assets/dd5b14ef-1ce8-44a5-bfc3-f4b95652756a)
I don't think a few gas cans are going to be enough to fill this station in the event of a loss of atmosphere.  I would recommend adding gas tank chambers like you see on the trade station.  They shouldn't have miners in them but they can have atmos marker tiles placed on them, these types of tanks hold an immense amount of gas. 
![image](https://github.com/user-attachments/assets/9a300b67-b693-4bb1-befb-06ac1ce9fe3c)
VS 
![image](https://github.com/user-attachments/assets/1fd69a92-4f9c-40a1-99a5-6c616c967c23)

MagnusCrowe commented:
> Thanks for the walk through.
> 
> I'm unsure about the trees in the garden space. But I'll play around with the area to see what else might be appropriate.
> 
> I originally had a much greater variety in lighting choices at spawn but ultimately abandoned that approach when most of the lights would be burnt out when the POI spawned in. But we have a new box of colored tubes thanks to another recent PR so that might be somewhat practical now.
> 
> This was the reason for creating the box of red/green bulbs for this PR, as otherwise I was finding that many of the dock lights were MIA.

Yeah the lights being busted is annoying, I think we should probably have a spawning flag to disable that on spawn.  The lights being busted by solar flares could also be toned down a little IMO.

MagnusCrowe commented:
Remove the anti-anomaly zone.

dustylens committed:
Magnus Changes

-Endarkened
-Engassened

MagnusCrowe commented:
> > Remove the anti-anomaly zone.
> 
> Sure.
> 
> > What about these areas are "Cargo"?
> 
> Easily change to dock if that seems more appropriate to you.
That area is a docking hub right? It seems more appropriate to me for it to be labeled as that.  It doesn't seem to have any connection to cargo at all.  It doesn't hold cargo, it doesn't sell cargo, it doesn't buy cargo.
May also consider calling it a lobby?  You can add a new station beacon for that label if you like.

> > Why does it have a gas condenser?
> 
> Largely unmapped tool due to the power draw that that was added as an additional utility piece for anyone who might have need of it. Which, it turns out, are a lot of visiting medics.
Yeah I understand it's a useful tool but why is it in the janitor closet?  It's a chemist machine.  I don't see the connection for why a janitor or someone doing cleaning would need a gas condenser.

> > I don't think a few gas cans are going to be enough to fill this station in the event of a loss of atmosphere. I would recommend adding gas tank chambers like you see on the trade station.
> 
> I think this was advised against when the station was originally being put together but persons siphoning atmosphere has largely subsided after the atmos price adjustments. Seems reasonable.

Siphoning isn't a really big deal if you don't include the miner like I specified.

MagnusCrowe commented:
> > May also consider calling it a lobby?
> 
> Could do. But as there's no lobby station beacon wouldn't that qualify as giving it a custom label? Just want to be sure that's good to go.

Yeah, the only reason you have a shops beacon is because I made it custom for the trade station.  Go nuts.  Now that I think of it, "Lobby" is a pretty useful beacon on Frontier.  Or "Hub".

> > Yeah I understand it's a useful tool but why is it in the janitor closet?
> 
> _gestures vaguely to the POI having a mandate to be small and spare_ General utility space. Reclaimer for materials, reclaimer for gas. Hook up a scrubber. If the MOM wants to move it elsewhere that's a choice. It's basically in the utility closet.

Considering it a "reclaimer" for gas is not a good argument because your station has no closed waste gas loop, all waste gases are vented, nothing can be reclaimed from it.  I'd be more inclined to believe your argument if it were directly connected to the waste pipe or at least had a canister port on the waste pipe to collect waste gases to be liquified after the canister fills.

> > Siphoning isn't a really big deal if you don't include the miner like I specified.
> 
> I believe I plumbed it up as desired.

You did.  Replace the walls facing into the engineering room with reinforced glass and grille. There should also be a selection of atmos signs that you could look at.  One of them should fit this area to indicate what the deal is with the little chambers.

dustylens committed:
Lobby update.

- Adds Lobby, MoM's office beacons.
- Slight map updates in accordance with maptainer feedback.

dustylens committed:
Strobe update.

dustylens committed:
Merge branch 'master' into NF-Dispatch-2

MagnusCrowe commented:
![image](https://github.com/user-attachments/assets/dfe9f90d-be5d-433a-8ab7-cc6ebe64ed28)
Include a box of normal light tubes and a box of normal light bulbs to mitigate the round start broken lights.

Your disposals pipe bounces garbage randomly when the conveyor isn't on.  I wonder if there isnt a better way to handle this.  Perhaps changing the angle of the disposals pipe or putting a windoor in.
![image](https://github.com/user-attachments/assets/6840ffec-82fb-4580-9cb6-ca6bc0debcd4)

MagnusCrowe commented:
These are comments about the new job:

MOM, really? Going with MOM? I swear to god if anyone calls me Mommy when I'm playing this job, I'm blaming you, not them.

I suggest making a locker specific to this role, making it indestructible and un-stealable (like the SR locker) so that we can put some job specific tools into it like maybe a full hypospray.

I strongly disagree with this job having access to the SR locker and everything else SR.  If necessary, it should have it's own access IMO.
![image](https://github.com/user-attachments/assets/c68e93bf-8ad1-4700-9dcd-04d625955f1c)


![image](https://github.com/user-attachments/assets/2f132ad5-5dea-479e-aac1-cef423e6245f)
Suggestions for the guidebook:

**Shift Objectives**
1) Pay attention to medical implant alerts and coordinate the paramedic response.  Ensure paramedics are routed to alerts and follow-up with patients to make sure they were cared for.
2) Coordinate medical training/mentorship by pairing interns with doctors.  You may personally train interns as long as it does not interfere with your other duties.
3) Manage the medical dispatch center's budget.  Compensate medical teams for their work when patients refuse to pay for service.  You are entitled to 20,000 spesos per hour worked from the dispatch center's budget.

**Medical Dispatch Budget** (not "your" ledger)
You will find a small medical dispatch budget at your Station Administration Console [insert picture].  This fund is replenished by taking a cut of each successful medical bounty claimed in this sector.

You may spend this money:
- To pay your own 20,000 speso per hour salary.
- To pay for station expenses. (Janitor services, hospitality services, upgrades, equipment, etc.)
- To compensate medical crews that heal patients who refuse to pay or cannot pay for medical services.

I would reword the last sentence in the guidebook entry to make it clearer that you can leave the dispatch center only as long as your responsibilities are being fulfilled.

dustylens committed:
Magnus requests

dustylens committed:
Merge branch 'NF-Dispatch-2' of https://github.com/dustylens/frontier-station-14 into NF-Dispatch-2

MagnusCrowe commented:
> > MOM, really? Going with MOM? I swear to god if anyone calls me Mommy when I'm playing this job, I'm blaming you, not them.
> 
> Remind me to add a bar of soap to Mom's loadout inventory.

Director of Care (DoC) is my suggestion.

Temoffy commented:
DoC's not bad, but a large majority of SS14 already has MOM as a head position.

arimah commented:
> DoC's not bad, but a large majority of SS14 already has MOM as a head position.

Really? I thought they mostly used Chief Medical Officer.

Temoffy commented:
Rouny's

: )

arimah commented:
RMC calls it MOM as a homage to Alien's MU-TH-UR/"mother" computer, no? I'd honestly much rather go with something less likely to result in weird RP. DoC seems like a good alternative, even if I'm not looking forward to the "what's up, DoC?" jokes. Alternatively stick with CMO, if this person is meant to be the premier authority on healthcare in the sector.

Temoffy commented:
I was attempting to joke about the xeno queen being a head role, but I failed.

I think any of the three names could work, but I see the potential for uncomfortable situations and comments for MoM.

MagnusCrowe commented:
> Some unique apparel and the like would be ideal. Both to keep the current CMO stuff in broad circulation but to also differentiate the role. However making visually interesting clothing in a timely manner is not something that I've had much success with. If a real spriter wanted to take on the task I'd be happy to do the backend work.

I was also thinking of giving them a medical ship voucher that they could use or give up to another player.

dvir001 committed:
Merge branch 'master' into NF-Dispatch-2

dvir001 committed:
Update lobby.ftl

dustylens committed:
Conflict II, son of conflict

dustylens committed:
Why is this even conflicting.

dustylens committed:
Merge branch 'master' into NF-Dispatch-2

whatston3 commented on Resources/Prototypes/_NF/Catalog/Fills/Paper/forms_medical.yml:
```diff
@@ -178,3 +180,72 @@
+      [color=#52B4E9]◥[bold]N[/bold]◣ [bold]Medical Operations Manager Payroll[/bold][/color]
+      [color=#52B4E9]‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾[/color]
+      [head=2]Medical Service Invoice[/head]
+
+      [head=3]Receipt of service for compensation. [/head]
+
```
```suggestion
```

I don't think this really adds anything.

whatston3 commented on Resources/Prototypes/_NF/Entities/Objects/Devices/pda.yml:
```diff
@@ -441,8 +441,52 @@
   components:
   - type: Pda
     id: NFJanitorIDCard
+    state: pda-janitor
```
```suggestion
```

whatston3 commented on Resources/Prototypes/_NF/Entities/Objects/Devices/pda.yml:
```diff
@@ -441,8 +441,52 @@
+    - BountyContractsCartridge
+    - AstroNavCartridge    
+    - LedgerCartridge
+    - MedTekCartridge #medtek preloaded. Should be ok.
```
```suggestion
    - MedTekCartridge #medtek preloaded. Should be ok.
  - type: Appearance
    appearanceDataInit:
      enum.PdaVisuals.PdaType:
        !type:String
        pda-mom
```

dustylens committed:
mom to doc

whatston3 commented on Resources/Prototypes/_NF/Entities/Structures/atm.yml:
```diff
@@ -227,6 +228,16 @@
+- type: entity
+  parent: BaseStationAdminBankATM
+  id: StationAdminBankATMMedical
+  description: Used to transfer funds to and from NFSD's bank account.
```
```suggestion
  description: Used to transfer funds to and from the medical bank account.
```

whatston3 reviewed (commented):
Partial review.

dustylens committed:
Update Resources/Prototypes/_NF/Entities/Structures/atm.yml

Co-authored-by: Whatstone <166147148+whatston3@users.noreply.github.com>

dustylens committed:
Update Resources/Locale/en-US/_NF/lobby/lobby.ftl

Co-authored-by: Whatstone <166147148+whatston3@users.noreply.github.com>

dustylens committed:
Update Resources/Prototypes/_NF/Catalog/Fills/Paper/forms_medical.yml

Co-authored-by: Whatstone <166147148+whatston3@users.noreply.github.com>

dustylens committed:
Update Resources/Prototypes/_NF/Entities/Objects/Devices/pda.yml

Co-authored-by: Whatstone <166147148+whatston3@users.noreply.github.com>

dustylens committed:
Update Resources/Prototypes/_NF/Catalog/Fills/Paper/forms_medical.yml

Co-authored-by: Whatstone <166147148+whatston3@users.noreply.github.com>

dustylens committed:
Update Resources/Prototypes/_NF/Catalog/Fills/Paper/forms_medical.yml

Co-authored-by: Whatstone <166147148+whatston3@users.noreply.github.com>

dustylens committed:
Update Resources/Prototypes/Catalog/Fills/Lockers/heads.yml

Co-authored-by: Whatstone <166147148+whatston3@users.noreply.github.com>

whatston3 commented on Resources/Prototypes/_NF/Roles/Jobs/Medical/doc.yml:
```diff
@@ -0,0 +1,38 @@
+  playTimeTracker: JobDirectorofCare
+  requirements:
+    - !type:OverallPlaytimeRequirement
+      time: 0 # 72000 # 20 hrs
```
```suggestion
      time: 72000 # 20 hrs
```
Shouldn't hurt.

whatston3 commented on Resources/Prototypes/_NF/PointsOfInterest/medical.yml:
```diff
@@ -0,0 +1,50 @@
+
+# Notes:
+# Medical ship garage, medical manager base and medical bounty staging area.
+- type: pointOfInterest
+  id: Medical
+  name: 'Medical Dispatch'
+  minimumDistance: 1650
+  maximumDistance: 3400
+  spawnGamePreset: [ NFAdventure, NFPirate ]
+  spawnGroup: Required
+  gridPath: /Maps/_NF/POI/medical.yml
+  addComponents:
+  - type: IFF
+    color: "#00b7eb"
+    readOnly: true
+  - type: Shuttle
+    angularDamping: 999999
+    linearDamping: 999999
+  - type: StationTransit
```
```suggestion
- type: pointOfInterest
  id: Medical
  parent: BasePOI
  name: 'Medical Dispatch'
  minimumDistance: 1650
  maximumDistance: 3400
  spawnGroup: Required
  gridPath: /Maps/_NF/POI/medical.yml
  addComponents:
  - type: IFF
    color: "#00b7eb"
  - type: StationTransit
```
A few of these fields can be inherited from the parent.
Not 100% sure these changes work, but should be roughly good.

whatston3 commented on Resources/Prototypes/_NF/Loadouts/role_loadouts.yml:
```diff
@@ -204,6 +204,27 @@
+  - ContractorImplanter
+  - ContractorFun
+  - ContractorTrinkets
+  - ContractorBureaucracy
```
```suggestion
  - ContractorBureaucracy
  - NFSpeciesSpecific
```
If you'd prefer the vox to have a medical mask by default, I can get you an NFMedicalSpeciesSpecific definition

whatston3 commented on Resources/Prototypes/_NF/Loadouts/doc_loadout_groups.yml:
```diff
@@ -0,0 +1,64 @@
+- type: loadoutGroup
+  id: DocShoes
+  name: loadout-group-contractor-shoes
+  loadouts:
+  - MercenaryClothingShoesBootsMercenary
+  - MercenaryClothingShoesBootsJack
+  - MercenaryClothingShoesBootsCombat
+  subgroups:
+  - PilotShoes
+  - ContractorShoes
+  fallbacks:
+  - MercenaryClothingShoesBootsMercenary
```
Might suggest a loadout with black, brown, white shoes, maybe laceups?  Dunno.

dustylens committed:
More Doc catches : )

whatston3 commented on Resources/Locale/en-US/_NF/job/job-description.ftl:
```diff
@@ -5,6 +5,7 @@ job-description-contractor = Fulfill any profitable contracts out on the frontie
 job-description-deputy = Catch criminals and enemies of the corporation, enforce the law, and ensure that the sector does not fall into disarray.
 job-description-nf-detective = Investigate crime scenes using forensic tools, ensure that the guilty party is found, and have a couple smokes.
 job-description-ertmailcarrier = Nothing stops the mail.
+job-description-medical-operations-manager = Provide guidance and direction for shift medics and work to safeguard the health of Nanotrasen personnel. 
```
```suggestion
job-description-director-of-care = Provide guidance and direction for shift medics and work to safeguard the health of Nanotrasen personnel. 
```
MOM-to-DOC

dustylens committed:
Update Resources/Prototypes/_NF/Roles/Jobs/Medical/doc.yml

Co-authored-by: Whatstone <166147148+whatston3@users.noreply.github.com>

dustylens committed:
Update Resources/Locale/en-US/_NF/job/job-description.ftl

Co-authored-by: Whatstone <166147148+whatston3@users.noreply.github.com>

dustylens committed:
Update Resources/Prototypes/_NF/Loadouts/role_loadouts.yml

Co-authored-by: Whatstone <166147148+whatston3@users.noreply.github.com>

dustylens committed:
Update Resources/Prototypes/_NF/PointsOfInterest/medical.yml

Co-authored-by: Whatstone <166147148+whatston3@users.noreply.github.com>

dustylens committed:
Merge branch 'master' into NF-Dispatch-2

whatston3 commented on Resources/Prototypes/_NF/Entities/Objects/Misc/rubber_stamp.yml:
```diff
@@ -81,3 +81,17 @@
     state: stamp-pal
+        
+- type: entity
+  name: Director of Care's rubber stamp
```
```suggestion
  name: director of care's rubber stamp
```
consistent casing

whatston3 commented on Resources/Prototypes/_NF/Entities/Objects/Misc/rubber_stamp.yml:
```diff
@@ -81,3 +81,17 @@
+  - type: Stamp
+    stampedName: stamp-component-stamped-name-doc
+    stampedColor: "#5b97bc"
+    stampState: "paper_stamp-nf-doctor"
```
```suggestion
    stampState: "paper_stamp-cmo"
```
I'd use this to be visually distinct from the doctor's stamp.

whatston3 reviewed (commented):
minor stamp suggestions

dustylens committed:
Update Resources/Prototypes/_NF/Entities/Objects/Misc/rubber_stamp.yml

Co-authored-by: Whatstone <166147148+whatston3@users.noreply.github.com>

dustylens committed:
Update Resources/Prototypes/_NF/Entities/Objects/Misc/rubber_stamp.yml

Co-authored-by: Whatstone <166147148+whatston3@users.noreply.github.com>

dustylens committed:
Adds flatpack, general cleanup.

dustylens committed:
Merge branch 'NF-Dispatch-2' of https://github.com/dustylens/frontier-station-14 into NF-Dispatch-2

dustylens committed:
Merge branch 'master' into NF-Dispatch-2

dustylens committed:
F... Fix?

dustylens committed:
Merge branch 'NF-Dispatch-2' of https://github.com/dustylens/frontier-station-14 into NF-Dispatch-2

dustylens committed:
I hate PDAs I hate PDAs I hate PDAs I hate PDAs I hate PDAs

dustylens committed:
Requested mapping changes, headset changes.

whatston3 committed:
Merge branch 'master' of https://github.com/new-frontiers-14/frontier-station-14 into NF-Dispatch-2

whatston3 committed:
reduce optional POIs to 6, add DOC encryption key

whatston3 committed:
Medical form tweaks, no common stamp "DO NOT MAP"

whatston3 committed:
"DirectorOfCare" caps, a million loadout changes

whatston3 reviewed (approved):
I think this is good.

Changes I made that could use a double check:
* Optional POI count reduced to 6 from 8 (could probably even be reduced to 5)
* Medical forms reworked slightly - DOC forms have a separate header colour for easier visual ID (they're still close)
* DOC loadouts changed - they've got a new encryption key with command, medical, and common access and a separate centcom key (only because they report directly to them - if this changes to where the SR is their boss, which I could see an argument for, fair enough)
* "DirectorofCare" changed to "DirectorOfCare"
* Gas mixture on Medical Dispatch fixed

Things that require some acknowledgement:
* AllAccess is probably fine if it's a role that's at the same level as the SR - it's a whitelisted role, and gross abuse should be an administrative issue - the fact that the whitelisting process lets that through is its own problem.
* Ditto for centcomm radio - since centcomm is their boss, I think it makes sense
* The station admin console flatpack is to allow for the DOC, in cases where it is reasonable, to run with a ship of their choice without needing DOC-specific versions of the ship.  You want to run a Spirit for whatever reason?  Power to you, you can hand out payments from your ship without issue.

This looks good to me.

whatston3 committed:
fallback->fallbacks

whatston3 committed:
Remove Frontier medical bounty redemption machines

Cheackraze commented:
https://github.com/new-frontiers-14/frontier-station-14/pull/2734 contains Medical Dispatch-specific holopads to be mapped at your descretion, 2 named ones, Reception and DoC's Office. Up to you how you use them, if they get merged before this one does in the next day or so

dustylens committed:
Small typo correction. Made name/species/occupation uniform across the, uh, forms.

dustylens committed:
Merge branch 'master' into NF-Dispatch-2

whatston3 commented on Resources/Prototypes/_NF/Catalog/Fills/Paper/forms_medical.yml:
```diff
@@ -133,12 +146,13 @@
 
           [bold]Additional Coverage:[/bold]
           \[  ] Towing: Coverage up to $10,000. Small shuttles only. ($2,000 premium)
+          \[  ] Towing: Coverage up to $20,000. Medium shuttles only. ($4,000 premium)
```
```suggestion
          \[  ] Towing: Coverage up to $20,000. Small and medium shuttles only. ($4,000 premium)
```
Reasonable.

dustylens committed:
Update Resources/Prototypes/_NF/Catalog/Fills/Paper/forms_medical.yml

Co-authored-by: Whatstone <166147148+whatston3@users.noreply.github.com>

dvir001 commented:
@whatston3 https://www.youtube.com/watch?v=Rn6GbXZCt-o

whatston3 commented:
@dvir001
Gave my piece here: https://github.com/new-frontiers-14/frontier-station-14/pull/2451#pullrequestreview-2561065165
Seems good to go on my side.
Wrote up an intro blurb with Dusty that seems fine to me.

Cheackraze commented:
i dont think we are at the limit of optional POIs anyhow since most of these being introduced lately are mandatory and shouldnt be bumping up against that limit. I also want to make a couple POIs to add a bit of variety in that specific spawn pool anyways.
